ms2query/sqlite.py

```diff
@@ -0,0 +1,136 @@
+"""
+Functions to create and retrieve information from sqlite files
+
+Author: Niek de Jonge
```

Comment:
```suggestion
```

Reply:
This is of course not to steal your authorship on this code ;)
It is more that we will otherwise loose track of who did which part of the code. So instead it would be better to add you name to `CITATION.cff` and the details of all code contributions will be nicely monitored by GitHub itself.

Reply:
Hahah, I get that. By now I have added it to citation.cff, so this will be added in the next pull request.